pkg/operator/controller/ingress/deployment.go

```diff
@@ -84,6 +84,11 @@ const (
 
 	RouterHAProxyConfigManager = "ROUTER_HAPROXY_CONFIG_MANAGER"
 
+	RouterHAProxyMaxDynamicServers             = "ROUTER_MAX_DYNAMIC_SERVERS"
+	RouterHAProxyMaxDynamicServersDefaultValue = 1
+
+	RouterHAProxyBlueprintRoutePoolSize = "ROUTER_BLUEPRINT_ROUTE_POOL_SIZE"
```

Comment:
These two env variables are not found in our haproxy.config template yet and I don't see a PR that adds them.  Does that matter?

Reply:
They are not used in the config template, they are only for the configuration of the router's DCM behavior.

Reply:
Ok, but it's strange that they'd need to be env variables in that case.

Reply:
The operator usually use environment variables rather than command-line switches for openshift-router, for which reason I asked Andrey to use environment variables in this case too (https://github.com/openshift/cluster-ingress-operator/pull/1159#discussion_r1811510003).